test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo docker build --force-rm -t pytrac/debian:stable .
+sudo docker run -rm -t -i  -v $PWD:/home/pytrac/data pytrac/debian:stable ./venv/bin/python ./setup.py test
```

Comment:
As the box is build exclusively you could use `ENTRYPOINT` for this. In that case you wouldn't have to care about the layout inside the container in your test script:

```docker
[...]
ENTRYPOINT ["/home/pytrac/venv/bin/python", "./setup.py", "test"]
```